tests/ops/test_cv_ops.py

```diff
@@ -40,6 +41,26 @@ def test_rotation_heisenberg(self, phi):
         )
         assert np.allclose(matrix, true_matrix)
 
+    @pytest.mark.parametrize("op,size", [
+            (cv.Squeezing(0.123, 0.456, wires=1), 3),
+            (cv.Rotation(0.123, wires=0), 3),
+            (cv.Displacement(0.123, 0.456, wires=0), 3),
+            (cv.Beamsplitter(0.456, 0.789, wires=[0, 2]), 5),
+            (cv.TwoModeSqueezing(2.532, 1.778, wires=[1, 2]), 5),
+            (cv.Interferometer(
+                np.array([[1, 1], [1, -1]]) * -1.j / np.sqrt(2.0),
+                wires=1), 5),
+        ])
+    def test_adjoint_cv_ops(self, op, size, tol):
+        op_d = op.adjoint()
+        op_heis = op._heisenberg_rep(op.parameters)
+        op_d_heis = op_d._heisenberg_rep(op_d.parameters)
+        res1 = np.dot(op_heis, op_d_heis)
+        res2 = np.dot(op_d_heis, op_heis)
+        np_testing.assert_allclose(res1, np.eye(size), atol=tol)
```

Comment:
This is new to me! What's the benefit of this instead of just `assert np.allclose()` ?

Reply:
This adds a nice error message when the test fails.

```python
>>> np.testing.assert_allclose([1, 2], [1, 3])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/chase/anaconda3/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 1527, in assert_allclose
    assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
  File "/home/chase/anaconda3/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 840, in assert_array_compare
    raise AssertionError(msg)
AssertionError:
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 1 / 2 (50%)
Max absolute difference: 1
Max relative difference: 0.33333333
 x: array([1, 2])
 y: array([1, 3])
```

vs

```python
>>> assert np.allclose([1, 2], [1, 3])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```